### PR TITLE
Update About and FAQ for Beta Release

### LINKED ISSUE #3538: Review FAQ and About for the beta
Just a placeholder to remind me to review the FAQ and About UIs to make sure they make sense for the beta.

### PR DESCRIPTION
**Pull request by ellisonbg**
Fixes #3538

### CODE REVIEW

**Comment by jasongrout** [comment]
I'm seeing similar test failures on #3559

File: packages/faq-extension/faq.md [hunk 1]
```diff
@@ -7,20 +8,31 @@
...
-users to arrange multiple Jupyter notebooks, text editors, terminals,
-output areas, etc. on a single page with multiple panels and tabs in
-one integrated application. The codebase and user-interface of JupyterLab
-is based on a flexible extension system that makes it easy to extend with new
-functionality.
+JupyterLab is a new user interface for Project Jupyter, allowing users to
+arrange multiple Jupyter notebooks, file editors, terminals, output areas,
+etc. on a single page with multiple panels and tabs in one integrated
+application. While JupyterLab looks like an IDE (Integrated Development
+Environment) it remains focused on the core Jupyter experience of
```
Comment (jasongrout) [inline packages/faq-extension/faq.md]: `), ` (missing comma)

File: packages/faq-extension/faq.md [hunk 2]
```diff
@@ -7,20 +8,31 @@
...
+interactive computing with data.
+
+In addition to the Jupyter Notebook, JupyterLab offers multiple models of
+interactive computing, including a scratchpad-like Code Console and the
+ability to attach kernels to any text file. JupyterLab has
+built-in support for many common file and outputs formats (CVS, PDF, Vega,
```
Comment (jasongrout) [inline packages/faq-extension/faq.md]: `file and output formats` (singularlize 'outputs')

File: packages/faq-extension/faq.md [hunk 3]
```diff
@@ -7,20 +8,31 @@
...
+Vega-Lite, Markdown, JSON, VDOM, PNG, JPEG, HTML).
```
Comment (jasongrout) [inline packages/faq-extension/faq.md]: `HTML, etc.)` - there are more, such as GIF, etc.

**Comment by jasongrout** [comment]
The text looks okay to me, modulo a few grammatical things I commented on inline.

File: packages/faq-extension/faq.md [hunk 4]
```diff
@@ -29,18 +41,21 @@ Notebook. Here are a few of them you may want to try out:
...
+for more detailed information about these and other features.
+
 ### How stable is JupyterLab?
 
-This beta version of JupyterLab will be characterized by:
+This Beta version of JupyterLab is ready for you to use! More specifically,
+it is characterized by:
 
-* Stable and featureful enough for daily usage by most Jupyter users.
+* Stable and featureful enough for daily usage by users.
```
Comment (rgbkrk) [inline packages/faq-extension/faq.md]: I think you could end this bullet with “daily usage”

File: packages/faq-extension/faq.md (hunk 1 above)
Comment (ellisonbg) [reply packages/faq-extension/faq.md]: fixed

File: packages/faq-extension/faq.md (hunk 2 above)
Comment (ellisonbg) [reply packages/faq-extension/faq.md]: fixed

File: packages/faq-extension/faq.md (hunk 3 above)
Comment (ellisonbg) [reply packages/faq-extension/faq.md]: fixed

File: packages/faq-extension/faq.md (hunk 4 above)
Comment (ellisonbg) [reply packages/faq-extension/faq.md]: fixed

Comment (ellisonbg) [reply packages/faq-extension/faq.md]: I guess we don't have to address how stable it is for non-users :-)

Comment (rgbkrk) [reply packages/faq-extension/faq.md]: It’s super stable for non-users! They shouldn’t have experience any problems!

**Comment by blink1073** [comment]
Thanks!

**Pull request merged**